### PR TITLE
Update function npm install

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: documentation/Functions.md [hunk 1]
```diff
@@ -281,19 +281,32 @@ If the tutorial should be tested on the console environment, you have to specify
...
-1. Path to the project where the dependencies from the package.json file are to be installed.
-2. Json-object: Name of a package, global or local installation, or array of npm arguments
-* (Optional) name of a package {"name": string }
-* (Optional) global or local installation. Default is local, therefore false {"global" : boolean }
-* (Optional) array of npm arguments {"args": string[]}
+You need to two parameters for this function.
+
+Example: npmInstall(1st-parameter, 2nd-parameter)
+
+1st-parameter: It is always needed. It is the path to the project where the dependencies from the package.json file are to be installed. It is of type string.
```
Comment (GuentherJulian) [inline documentation/Functions.md:290]: I think we don't need to mention when a parameter is explicitly required. Just mention when a parameter is optional

File: documentation/Functions.md [hunk 2]
```diff
@@ -281,19 +281,32 @@ If the tutorial should be tested on the console environment, you have to specify
...
+2nd-parameter: It is optional. It is the array of Json-objects which contains Name of a package, global or local installation, or array of npm arguments.
```
Comment (GuentherJulian) [inline documentation/Functions.md:291]: JSON in uppercase. name with leading lowercase.

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md [hunk 3]
```diff
@@ -281,19 +281,32 @@ If the tutorial should be tested on the console environment, you have to specify
...
 
 ### npmInstall <a name="npmInstall"></a>
+npmInstall is used for installing packages from node package mangager, for example used in nodejs or angular projects.
 #### parameter 
-1. Path to the project where the dependencies from the package.json file are to be installed.
-2. Json-object: Name of a package, global or local installation, or array of npm arguments
-* (Optional) name of a package {"name": string }
-* (Optional) global or local installation. Default is local, therefore false {"global" : boolean }
-* (Optional) array of npm arguments {"args": string[]}
+You need to two parameters for this function.
```
[the remark this answers is not in the document]
Comment (EduardKrieger) [reply documentation/Functions.md:286]: Try to use the same wording for every function. I would prefer if you take: "You need to pass two parameters for this function"

File: documentation/Functions.md [hunk 4]
```diff
@@ -281,19 +281,44 @@ If the tutorial should be tested on the console environment, you have to specify
 ***
 
 ### npmInstall <a name="npmInstall"></a>
+npmInstall is used for installing packages from node package mangager, for example used in nodejs or angular projects.
```
Comment (GuentherJulian) [inline documentation/Functions.md:284]: Node.js or Angular

File: documentation/Functions.md [hunk 5]
```diff
@@ -281,19 +281,44 @@ If the tutorial should be tested on the console environment, you have to specify
...
+* npmInstall("jump-the-queue/angular", {"name": "@angular/cli", "global": true, "args": ["--save-dev"]})
 will run 'npm install -g --save-dev @angular/cli' in the directory 'jump-the-queue/angular'.
 
-npmInstall("my-thai-star/angular")
+* npmInstall("my-thai-star/angular")
 will run 'npm install' in the directory 'my-thai-star/angular'
 
+Note: 
+1. Existing project is required to execute this command.
+2. The command for execution will be generated by katacoda runner, so user will have to execute this command manually.
```
Comment (GuentherJulian) [inline documentation/Functions.md:320]: Katacoda

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 4 above)
Comment (dixyushi) [reply documentation/Functions.md:284]: Changed!!

File: documentation/Functions.md (hunk 5 above)
Comment (dixyushi) [reply documentation/Functions.md:320]: Changed!!

**Pull request merged**